src/components/input/Input.vue

```diff
@@ -109,9 +109,7 @@
         },
         data() {
             return {
-                // This is necessary because, for some reason, textarea
-                // returns undefined if there's no v-model or value.
```

Comment:
Let me test this again before merging, it was a pretty weird behavior if I remeber correctly.